CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.0.0-rc.3 (2018-11-20)
+
+- Fixed datepicker and timepicker components to be able to set the disabled state in reactive forms. [blackbaud/skyux-datetime#4](https://github.com/blackbaud/skyux-datetime/pull/4)
```

Comment:
How about `Fixed datepicker and timepicker components to set the disabled state in reactive forms....`?